dev/datasources.json

```diff
@@ -20,5 +20,16 @@
     "tags":[
       "help-needed", "2D", "plant-phenotyping", "agriculture", "climate-change-and-agriculture"
     ]
+  },
+  "data-002":{
+    "task":["classification"],
+    "domain":["oceanography"],
+    "datasource":"https://github.com/scotthosking/intake-plankton",
```

Comment:
Do we want this to point to the catalog yaml file?

Reply:
Yes, we should do. I was thinking that we might standardise the yaml file name but it would be safest to name it. Alternatively we could include a scivision.conf to do the plumbing for us